src/pages/index.tsx

```diff
@@ -103,13 +103,13 @@ const features = [
     description: (
       <>
         <p>
-          <b>Codegen.</b> Generate tests by recording your actions. Save them into any language.
+          <b><a href="/docs/codegen">Codegen.</a></b> Generate tests by recording your actions. Save them into any language.
```

Comment:
these should be relative URLs so that they work from the language ports as well:
https://playwright.dev/python/
https://playwright.dev/dotnet/
https://playwright.dev/java/
https://playwright.dev/